qiskit/primitives/base_estimator.py

```diff
@@ -143,6 +150,14 @@ def __init__(
         """
         self._circuits = tuple(circuits)
         self._observables = tuple(observables)
+        self._circuit_ids = (
+            self._circuit_ids if self._circuit_ids else [circuit.name for circuit in self._circuits]
```

Comment:
👍 Using `circuit.name` seems consistent with what we do we `Result` objects, so makes sense.

Reply:
Thanks. I changed to use name in `QuantumCircuit`, but I forgot to change the variable name. I will fix this.